ui-admin/src/portal/siteContent/HtmlPageEditView.tsx

```diff
@@ -47,11 +25,20 @@ const HtmlPageView = ({ htmlPage, updatePage, readOnly }: HtmlPageViewProps) =>
   }
 
   return <div>
+    <div className="col-md-12 my-2" style={{ backgroundColor: '#eee' }}>
+      <Button variant="secondary"
+        aria-label={'Insert a blank section'}
+        tooltip={'Insert a blank section'}
+        disabled={readOnly}
+        onClick={() => insertNewSection(0, { id: '', sectionType: 'HERO_WITH_IMAGE' })}>
```

Comment:
What role does the JS template code associated with the `sectionType` play? Just trying to understand how this works since the code section is blank when you insert the section.

Reply:
Good question, this doesn't play with those template configs yet. But that would be a good next step to make the newly-added sections more useful, so you could have a pre-populated template in the section editor instead of having to copy-paste.